agent/main.py

```diff
@@ -0,0 +1,85 @@
+import sys
+import os
+
+from lib.agent import Agent
+from lib.event import Event
+from lib.controlloop import run_loop
+
+if len(sys.argv) < 3:
+    print("Usage: python main.py <working_directory> <task>")
+    sys.exit(1)
+
+working_directory = sys.argv[1]
```

Comment:
@rbren Maybe `argparse` is more scalable and readable.

Reply:
Yeah we should definitely switch to something more robust!